components/hacbs/integration-service/overlays/dev/apiexport.yaml

```diff
@@ -1,32 +1,34 @@
+---
 apiVersion: apis.kcp.dev/v1alpha1
 kind: APIExport
 metadata:
-  name: integration-service
+  name: integration-service-integration-service
```

Comment:
@dirgim @Michkov It is strange, why is it changed to `integration-service-integration-service`?

Reply:
I think that this is fixing it -> https://github.com/redhat-appstudio/integration-service/pull/55/files#diff-4f3c77575ecb65d2c44e33b1610212bd5de4d11c2ebdeb718d2b5c44ed075dc1L9

Reply:
But the name was not changed to this style in https://github.com/redhat-appstudio/infra-deployments/pull/831/files

Reply:
There was bug in `generate-apiexport-overlays.sh` - https://github.com/redhat-appstudio/infra-deployments/pull/823, so maybe due to it the update was skipped. Let's see how the PR will look like after merge of https://github.com/redhat-appstudio/integration-service/pull/55

Reply:
Looks like that fixed it